scripts/mumble-version.py

```diff
@@ -100,44 +100,40 @@ def readProjectVersion():
 
 def main():
     parser = argparse.ArgumentParser()
+    parser.add_argument('-f', '--format', choices = ['full', 'version', 'suffix'], default='full', help = 'Output format')
     parser.add_argument('-n', '--newline', action = "store_true", help = 'Break line after printing version')
-    parser.add_argument('-p', '--project', action = "store_true", help = 'Print CMake project version')
+    parser.add_argument('-r', '--revision', type = int, default = 1, help = 'Revision, only used for \'beta\' and \'rc\'')
+    parser.add_argument('-t', '--type', choices = ['snapshot', 'beta', 'rc', 'stable'], default = 'snapshot', help = 'Release type, determines the suffix')
     args = parser.parse_args()
 
     if args.newline:
         end = None
     else:
         end = ''
 
-    if args.project:
-        projectVersion = readProjectVersion()
-        print(projectVersion, end = end)
-        return
-
-    # Get all tags associated with the latest commit
-    latestCommitTags = [x for x in cmd(['git', 'tag', '--points-at', 'HEAD']).split("\n") if x]
+    version = readProjectVersion()
 
-    if len(latestCommitTags) > 1:
-        raise RuntimeError("Encountered commit with multiple tags: %s" % latestCommitTags)
+    if args.format == 'version':
+        print(version, end = end)
+        return
 
-    if len(latestCommitTags) == 1:
-        # Most recent commit is tagged -> this is a tagged release version
-        # Use the tag as the version-string
-        version = latestCommitTags[0]
-    else:
-        # This is a snapshot (i.e. built from a non-tagged commit)
+    suffix = ''
 
+    if args.type == 'rc' or args.type == 'beta':
+        suffix = '{0}{1}'.format(args.type, args.revision)
```

Comment:
```suggestion
        suffix = '-{0}{1}'.format(args.type, args.revision)
```
I think we need this change in order to get the results as mentioned in your first commit.

Reply:
Oops, you're right.